js/packages/web/src/views/artCreate/infoStep.tsx

```diff
@@ -46,22 +46,17 @@ export const InfoStep = (props: {
               <h3>Name</h3>
               <Input
                 autoFocus
-                placeholder="Max 20 characters (fewer if using emojis)"
+                placeholder="Max 32 characters (fewer if using non-latin characters)"
                 allowClear
                 value={props.attributes.name}
                 onChange={info => {
-                  props.setAttributes({
```

Comment:
Do we invalidate the form if the length is wrong? It’s important that the users never make it to the mint step with an invalid title.

Reply:
The way it is now, it just isn't possible to update the input field state with anything that isn't valid (the onchange prevents that).  I know antd is doing some additional validation though so will look into how that works. 

Also, we could add a disabled property on the submit button that is set to true if for some reason the form values are invalid -- thoughts?

cc @kespinola

Reply:
That'd be good, my concern with just not setting it is that users won't notice their title wasn't applied.  Want to avoid having to tell users to burn items because the title was wrong.

Reply:
hmm, how would we catch it though?  no way for us to know what their desired title is right?

basically right now if they try typing in or pasting anything that ends up over 32 bytes, there's just no response.  maybe we add in an else block in the onchange that makes an error message display?

Reply:
yeah, that'd be good